core/test/acceptance/old/admin/utils.js

```diff
@@ -115,12 +115,12 @@ module.exports = {
         return testUtils.API.doAuth(`${API_URL}session/`, ...args);
     },
 
-    getValidAdminToken(endpoint) {
+    getValidAdminToken(audience) {
         const jwt = require('jsonwebtoken');
         const JWT_OPTIONS = {
             algorithm: 'HS256',
             expiresIn: '5m',
-            audience: endpoint
+            audience: audience
```

Comment:
@allouis FYI this is the v2 admin utility. No need to pass /admin/v2/ from tests.